src/Service/SF1500/SF1500.php

```diff
@@ -397,88 +388,67 @@ public function getEnhedNavnOgOverordnetOrganisationsId($organisationEnhedsId, $
 
     /**
      * Fetches bruger and organisation funktions id for managers from user id.
-     * Returns null if no manager exists.
+     * Returns empty array if no manager exists.
+     * @throws SF1500Exception|SAMLTokenException
      */
-    public function getManagerBrugerAndFunktionsIdFromUserId($userId, $managerFunktionsTypeId)
+    public function getManagerBrugerAndFunktionsIdFromUserId($userId, $managerFunktionsTypeId): array
     {
-        $token = $this->fetchSAMLToken();
-
         $organisationFunktionsId = $this->getOrganisationFunktionerFromUserId($userId);
 
-        if (null === $organisationFunktionsId) {
-            throw new SF1500Exception(sprintf('Could not find any organisation funktioner for user %s', $userId));
+        if (empty($organisationFunktionsId)) {
+            throw new SF1500Exception(sprintf('Cannot find any organisation funktioner for user %s', $userId));
         }
 
         $managers = [];
-        if (is_array($organisationFunktionsId)) {
-            foreach ($organisationFunktionsId as $id) {
-                $managerInfo = $this->getManagerBrugerAndFunktionsIdFromFunktionsId($id, $managerFunktionsTypeId, $token);
 
-                if (null !== $managerInfo) {
-                    $managers = array_merge($managers, $managerInfo);
-                }
-            }
-        } else {
-            $managerInfo = $this->getManagerBrugerAndFunktionsIdFromFunktionsId($organisationFunktionsId, $managerFunktionsTypeId, $token);
+        foreach ($organisationFunktionsId as $id) {
+            $managerInfo = $this->getManagerBrugerAndFunktionsIdFromFunktionsId($id, $managerFunktionsTypeId);
 
-            if (null !== $managerInfo) {
-                $managers = $managerInfo;
+            if (!empty($managerInfo)) {
+                $managers = array_merge($managers, $managerInfo);
```

Comment:
Use `$managers[] = $managerInfo;` here. It's more efficient than `array_merge`.

Reply:
Fixed.